static/js/publisher/release/revisionsList.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+export default class RevisionsList extends Component {
+  renderRows(revisions) {
+    return revisions.map((revision) => {
+      const uploadDate = moment(revision.timestamp);
+
+      return (
+        <tr key={revision.revision}>
+          <td>#{ revision.revision }</td>
+          <td>{ revision.version }</td>
+          <td>{ revision.arch }</td>
+          <td>{ revision.channels.join(", ") }</td>
+          <td>{ uploadDate.format("YYYY-MM-DD HH:mm") }</td>
+          <td className="u-align--right">{ uploadDate.fromNow() }</td>
+        </tr>
+      );
+    });
+  }
+
+  render() {
+    return (
+      <table>
```

Comment:
We should use the vanilla-react sortable table here, I think

Reply:
I tried to use Table from vanilla-framework-react and found 2 issues that make it really hard to use, so went with simple approach instead. The only sorting we need for now is by revision number I guess, so should be fine like this for a while.